posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -153,15 +153,15 @@ spec:
         name: secure-forward
 ```
 --
-. Create the instance inside your OpenShift cluster:
+. Create the Log Forwarding API instance inside your OpenShift cluster:
```

Comment:
is that accurate?

Reply:
`Log Forwarding` instead.
The _Log Forwarding API_ more refers to the the configuration value that are possible in the yaml file.

Reply:
updated